docs/hydra/sdk/api.md

```diff
@@ -8037,10 +8037,6 @@ token was not granted one of the scopes, the result of active will be false.
 }
 ```
 
-<aside class="warning">
```

Comment:
This is generated from code [here](https://github.com/ory/hydra/blob/master/oauth2/handler.go#L390-L392), so it needs to be fixed there.